emarchik/lesson_4/ansible/ansible_first_task.yml

```diff
@@ -0,0 +1,27 @@
+---
+########################################
+##### First Ansible Task: 25.02.19 #####
+########################################
+
+- name: Asible First Task
+  hosts: workstation
+  tags:
+    - first_task
+
+  tasks:
+  - name: For ReadHat/CentOS
+    remote_user: agarim
+    shell: |
+      hostname
+      hostname -I
+      sudo yum update -y
+    when: "ansible_os_family == 'RedHat'"
+
+  - name: For Debian/Ubuntu
+    remote_user: agarim
+    shell: |
+      hostname
+      hostname -I
+      sudo apt-get update
```

Comment:
Do not use for update shell command, need use modules (apt/yum)

Reply:
I understand.